src/components/Badge.tsx

```diff
@@ -30,14 +30,12 @@ interface BadgeProps {
   prefix?: string
 }
 
-let Badge = ({ content, className, amount, prefix }: BadgeProps) => {
-  return (
```

Comment:
I know you prefer to use the `return` keyword instead of the shorter arrow function syntax, so I only removed it from "dumb" components (dump == they do not have any logic, they simply return JSX) and kept it in utility and other functions. Let me know if you have a problem with that.

Reply:
Perfectly fine!